src/content/docs/style-guide/images/image-annotations.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Best practices for using image annotations
+contentType: page
+template: basicDoc
+topics:
+  - Basic style guide
+  - Writing guidelines
+---
+
+import styleguideExampleCalloutBox from 'images/style-guide_screenshot-full_example_callout_box.png'
+
+import apisanddataNerdGraphDashboardEntity from 'images/apis-and-data_screenshot-full_NerdGraph-dashboard-entity.png'
+
+import styleguideExampleArrow from 'images/style-guide_screenshot-full_example_arrow.png'
+
+Image annotations are text and shapes in screen captures or diagrams that help to highlight information and focus attention on the most important aspects of the screenshot. Avoid the use of text inside a screenshot, use [captions](/docs/style-guide/images/screenshots-images/#captions) instead.
+
+If you need to use image annotations:
+
+* Be concise.
+* Make sure annotations are easy to distinguish from UI text. For example, you may need to offset your boxes so they don't look like UI elements.
+* Annotations aren't translated, so use them sparingly.
+* Don't use an annotation at the top that acts like a title. Use a caption.
+* Use our [New Relic SnagIt theme](https://github.com/newrelic/newrelic-doc-tools) (link only available to New Relic employees) to keep your annotations consistent. 
+
```

Comment:
I would add an h2 here: Install the theme.

Then, you could have an introductory sentence like this: We have a SnagIt theme that helps us use the same conventions for annotations.

Reply:
Yes, added now, sorry.